### PR TITLE
feat: change zips from DocumentRepresentation to ArchiveRepresentation

### PR DESCRIPTION
**Pull request by BalduinLandolt**

### CODE REVIEW

**Comment by BalduinLandolt** [comment]
sorry about the overly numernous changes in the JSON and XML. Auto-formatting did its thing...

**Review by irinaschubert — APPROVED** [review]
LGTM, just a few questions regarding the example files. These should/can probably be answered best by @gautschr ?:
- Are rosetta.xml and rosetta.json (and Rosetta.zip) necessary inside the Documents folder? They seem to duplicate the files in the root folder of the repository.
- In the 3D folder there are other file types (Stygimoloch.png, Stygimoloch.zip) than 3D files. Is this OK so?
- In the Images folder there are other file types (Rosetta.zip, RosettaCategoryList.csv, TestprojectRosetta.pdf) than images. Is this OK so?

File: rosetta.xml [hunk 1]
```diff
@@ -1,10 +1,6 @@
 <?xml version='1.0' encoding='utf-8'?>
 
-<knora xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xmlns="https://dasch.swiss/schema"
-       xsi:schemaLocation="https://dasch.swiss/schema https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/knora/dsplib/schemas/data.xsd"
-       shortcode="082E"
-       default-ontology="rosetta">
+<knora xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="https://dasch.swiss/schema" xsi:schemaLocation="https://dasch.swiss/schema https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/knora/dsplib/schemas/data.xsd" shortcode="082E" default-ontology="rosetta">
 
     <!-- :permissions see https://docs.knora.org/03-apis/api-v2/reading-user-permissions/ -->
```
Comment (irinaschubert) [inline rosetta.xml:5]:
```suggestion
    <!-- :permissions see https://docs-api.dasch.swiss/02-knora-ontologies/knora-base/#permissions -->
```

**Comment by gautschr** [comment]
@irinaschubert concerning the files:
- rosetta.xml and rosetta.json should stay in the Documents folder. They are part of the zip file.
- since we can't properly deal with 3D images so far, the current workaround is the upload of the zip-file for such cases. The png-file can be eliminated.
- rosetta.zip, RosettaCategoryList.csv and RosettaTestProject.pdf can be delted in the Images-folder. They belong to the Documents-folder.

I'll clean this in a next step.

**Review by gautschr — APPROVED** [review]
Everything works properly - thank you!

**Pull request merged**